packages/blockchain-wallet-v4-frontend/src/scenes/DebitCard/template.tsx

```diff
@@ -45,45 +47,49 @@ const LinkHereBtn = () => (
   </div>
 )
 
-const DebitCard = () => (
-  <Wrapper>
-    <SceneHeader>
-      <HeaderTextWrapper>
-        <IconBackground>
-          <Icon name='credit-card-sb' color='blue600' size='26px' />
-        </IconBackground>
-        <SceneHeaderText>
-          <FormattedMessage id='copy.debit_card' defaultMessage='Debit Card' />
-        </SceneHeaderText>
-      </HeaderTextWrapper>
-    </SceneHeader>
-    <SceneSubHeaderText>
-      <FormattedMessage
-        id='scenes.debit_card.intro.subheader'
-        defaultMessage='Taking crypto into the physical world.'
-      />
-    </SceneSubHeaderText>
-    <IntroCardWrapper>
-      <Image name='intro-card' />
-
-      <SceneHeaderText>
-        <FormattedMessage
-          id='scenes.debit_card.intro.content.header'
-          defaultMessage='Your Gateway To The Blockchain Debit Card'
-        />
-      </SceneHeaderText>
+const DebitCard = (props: Props) => {
+  const openOrderMyCard = () =>
+    props.modalActions.showModal(ModalName.ORDER_MY_CARD, { origin: 'DebitCardPage' })
```

Comment:
Nice one! I would rename it to `handleOpenMyCard`, as I prefer to have `handle` on functions

Reply:
Renamed